### DISCUSSION TITLE
Some Filipino words are too informal! Please Help... It's boring

### DISCUSSION THREAD

**Discussion by Killer8Hyper** [monkeytypegame/monkeytype#1103] [Ideas] [answered]
First of all, YUNG is the informal word of IYONG. If with apostrophe 'YONG or 'YUNG 
ung < I want you to delete that one (ung)
ninyo = nyo

I want to delete those two words because they annoy and confuse me when I'm typing. Also, it's because there are Filipino words with two or three meanings in one word such as ganun, syempre, nyan nung and among others.

If possible please improve some Filipino words because in 10FF I don't find any wrong words and it's fun to type those words.

**Comment by unrealapex** [comment, marked as answer]
I am sure changes to the words will come but it might be marked as a low priority task because its not exactly a bug or a major feature. If you want your changes implemented as quickly as possible, try submitted a pull request with the changes to the words you've included. 

> I want to delete those two words because they annoy and confuse me when I'm typing.

I'm not certain about this because it is more of an opinion. 
If you are unhappy with it, feel free to fork the project and change it to your heart's desire.

**Reply by Killer8Hyper** [reply]
Thanks, but for now I can't do those things. And thanks again the one who created those common Filipino words, reply to me that he'll update those things ASAP.

**Reply by unrealapex** [reply]
Created an issue on it

**Reply by Miodec** [reply]
Any language tweaks other than English I can't do myself and rely on other native speakers - if there are any mistakes or a better list available, make a pull request or provide the list.
